portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,25 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+
```

Comment:
Particularly when working with others in a shared code base, you should not make whitespace changes.